pilots/10_things_to_know_about_pilots.Rmd

```diff
@@ -0,0 +1,189 @@
+
+Using the recommendations from DeclareDesign (2019) and equation 3.6 in Gerber and Green (2012), we provide sample code based on a hypothetical pilot study. In the code below, we assume we have already conducted a pilot study and have calculated the standard deviation of a key outcome measure for both the control and treatment groups. Next, we use these estimates to calculate MDEs for different possible sample sizes, in order to inform our target sample size for a future main study.
+
+```{r echo = T, warning = F, message = F}
+require(tidyverse)
+
+#Standard deviations of outcome measure for treatment & control groups calculated from pilot study results
+#The ones provided are for purposes of demonstration
+#We assume one treatment and control group--you will need to adjust to study specifics
+sd_control <- 1.5
+sd_treat <- 1.2
+
+#MDE calculation for various possible sample sizes
+N <- seq(from=500, to=3000, by=100)
+MDE <- vector()
+for (i in 1:length(N)){
+  #We assume an equal number of participants in treatment and control--you will need to adjust to study specifics
+  MDE[i] <- 2.8 * sqrt((sd_control^2/(N[i]/2)) + (sd_treat^2/(N[i]/2))) #Formula (equation 3.6) for estimated standard error of ATE in Gerber and Green (2012)
+}
+
+#Visualizing results to identify target sample size for main study
+mde_data <- as.data.frame(cbind(N, MDE))
+mde_plot <- ggplot(mde_data, aes(x=N, y=MDE)) + geom_point() +
+  xlab("Sample Size") + ylab("MDE") + ggtitle("Minimum Detectable Effect (MDE) for Main Study by Sample Size") + theme_bw() +
+  geom_hline(yintercept = .2, lty = "dashed")
+mde_plot
+```
+
+Based on the hypothetical standard deviations in the control and treatment groups from the pilot study, the main study would need a sample size of approximately 1500 to ensure that effect sizes as small as 0.2 are detectable. Note that while calculating the MDE based on pilot results is straightforward, determining how small an MDE should be is more subjective, and should be informed by theory and prior work.
+
+It's important to remember that your MDE calculations are based on the design of the pilot and the specific outcome measures used, including, for example, whether the associated estimate requires you to subset your data. Thus, you may need to perform MDE calculations for each estimand of interest to determine what sample sizes are needed and which hypotheses can be addressed with sufficient power given your experimental design. Keep in mind that as you use pilot results to refine your study design, including treatments and outcomes, the relevant standard deviations of the outcome measures and the resultant MDE calculations may change. Another quick pilot never hurts.

```

Comment:
What if the research team only has resources to run one pilot? Is the recommendation to make changes based on the first one, and then move forward with full study? Or to take resources away from the full study to run a 2nd pilot before proceeding (even if that means a smaller sample size in the full study)?